app/config/sculpin_site.yml

```
@@ -58,6 +58,9 @@ subspecies:
     phpclasses:
         common: "PHP Classes"
         latin: "decuriae"
+    phpers:
+        common: "PHPers Summit"
+        latin: "conventus"
```

How about going with the same "top"/"meeting" meaning as in English?
```suggestion
        latin: "summum"
```